controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -717,6 +722,13 @@ func (r *BareMetalHostReconciler) actionInspecting(prov provisioner.Provisioner,
 		return result
 	}
 
+	// Get firmware settings and manage resources
+	err = r.storeHostFirmwareSettings(prov, info)
+	if err != nil {
+		// log but continue, failure to get settings is not an error
+		info.log.Info("could not get firmware settings for host")
```

Comment:
Maybe log the error? Otherwise it's probably lost.

Reply:
Fixed.